ansible/README.md

```diff
@@ -43,7 +43,8 @@ These playbooks are available to you:
   - **jenkins/linter.yml**: Sets up the code linters (flavour of a worker).
 
   - **create-webhost.yml**: Configures the server(s) that host nodejs.org,
-                            iojs.org and dist.libuv.org among other things.
+                            iojs.org and dist.libuv.org among other things
+                            (only works on Ubuntu boxes).
```

Comment:
I thought it was somewhat self-explanatory seeing how the playbook only adds those hosts from the inventory.